mcc/odin/odin/instance_restore_test.go

```diff
@@ -97,10 +85,10 @@ func TestGetRestoreDBInstanceFromDBSnapshotInput(t *testing.T) {
 						)
 					}
 				} else {
-					if !equalsRestoreDBInstanceFromDBSnapshotInput(
+					if diff := deep.Equal(
```

Comment:
It is initializing the `diff` variable to the result of a function called `deep.Equal` that compares structures field by field. Right afterwards checks that diff is true, so the structures really are different, to report a test error.